rules/windows/execution_suspicious_powershell_imgload.toml

```
@@ -63,56 +63,51 @@ Attackers can use PowerShell without having to execute `PowerShell.exe` directly
 risk_score = 47
 rule_id = "852c1f19-68e8-43a6-9dce-340771fe1be3"
 severity = "medium"
-tags = ["Domain: Endpoint", "OS: Windows", "Use Case: Threat Detection", "Tactic: Execution", "Resources: Investigation Guide", "Data Source: Elastic Endgame", "Data Source: Elastic Defend"]
+tags = ["Domain: Endpoint",
+        "OS: Windows",
+        "Use Case: Threat Detection",
+        "Tactic: Execution",
+        "Resources: Investigation Guide",
+        "Data Source: Elastic Defend"
+        ]
 timestamp_override = "event.ingested"
-type = "eql"
+type = "new_terms"
 
 query = '''
-library where host.os.type == "windows" and
-  dll.name : ("System.Management.Automation.ni.dll", "System.Management.Automation.dll") and
-  not 
-  (
-    /* MS Signed Binaries */
-    (
-      process.code_signature.subject_name : (
-        "Microsoft Windows",
-        "Microsoft Dynamic Code Publisher",
-        "Microsoft Corporation"
-      ) and process.code_signature.trusted == true and not process.name : ("rundll32.exe", "regsvr32.exe")
-    ) or
-
-    /* Signed Executables from the Program Files folder */
-    (
-      process.executable : (
-        "?:\\Program Files (x86)\\*.exe",
-        "?:\\Program Files\\*.exe"
-      ) and process.code_signature.trusted == true
-    ) or
-
-    /* Lenovo */
-    (
-      process.executable : (
-        "?:\\Windows\\Lenovo\\*.exe"
-      ) and (process.code_signature.subject_name : "Lenovo" and process.code_signature.trusted == true) 
-    )
-  )
+host.os.type:windows and event.category:library and 
+
+dll.name:("System.Management.Automation.dll" or "System.Management.Automation.ni.dll") and 
+
+not (process.code_signature.subject_name:("Microsoft Corporation" or "Microsoft Dynamic Code Publisher" or "Microsoft Windows") and process.code_signature.trusted:true and not process.name.caseless:("regsvr32.exe" or "rundll32.exe")) and 
+
+not (process.executable.caseless:(?\:\\\\Program?Files?\(x86\)\\\\*.exe or ?\:\\\\Program?Files\\\\*.exe) and process.code_signature.trusted:true) and 
+
+not (process.executable.caseless:?\:\\\\Windows\\\\Lenovo\\\\*.exe and process.code_signature.subject_name:"Lenovo" and 
+  process.code_signature.trusted:true) and not process.executable.caseless : "C:\\Windows\\System32\\powershell.exe"
```

nit, unless there was a reason for the spacing?

```suggestion
  dll.name:("System.Management.Automation.dll" or "System.Management.Automation.ni.dll") and 
  not (process.code_signature.subject_name:("Microsoft Corporation" or "Microsoft Dynamic Code Publisher" or "Microsoft Windows") and process.code_signature.trusted:true and not process.name.caseless:("regsvr32.exe" or "rundll32.exe")) and 
  not (process.executable.caseless:(?\:\\\\Program?Files?\(x86\)\\\\*.exe or ?\:\\\\Program?Files\\\\*.exe) and process.code_signature.trusted:true) and 
  not (process.executable.caseless:?\:\\\\Windows\\\\Lenovo\\\\*.exe and process.code_signature.subject_name:"Lenovo" and 
  process.code_signature.trusted:true) and not process.executable.caseless : "C:\\Windows\\System32\\powershell.exe"
```